### DISCUSSION TITLE
news feed date is shown on two line. How to fix it?

### DISCUSSION THREAD

**Discussion by shashikantilager** [alshedivat/al-folio#1871] [Q&A] [answered]
Dear all,

The news entry on my main page shows the "date" spread on two lines instead of one inline entry (Month, day "\n"  year). How can we fix this?

**Comment by george-gca** [comment, marked as answer]
I don't know why this is happening for you, but to fix maybe you can set the number of columns that will be used for the date. Change [this line here](https://github.com/alshedivat/al-folio/blob/a348bee33997ca72beae217d1be632123224c114/_includes/news.html#L14) for this:

Before:
```html
<th scope="row">{{ item.date | date: "%b %-d, %Y" }}</th>
```

After:
```html
<th scope="row" class="col-sm-2">{{ item.date | date: "%b %-d, %Y" }}</th>
```

**Comment by shashikantilager** [comment]
Thanks, George; when I use the two columns for a date as you suggested, it works only correctly in Chrome browser where it does not have scroll enabled (not sure why).
On Safari and mobile browsers,  this two-column date entry pushes the news item to the extreme right where only two three words per line are present making the rendering ugly.

My question is 
1. Why is scroll enabled only on Safari and mobile browsers?
2. Any solution that works in all browsers?
